tsdb/index/postings.go

```diff
@@ -49,46 +50,63 @@ var ensureOrderBatchPool = sync.Pool{
 	},
 }
 
+const shardCount = 256
+
+type memPostingMap struct {
+	mtx    *sync.RWMutex
+	series map[string]map[string][]storage.SeriesRef
```

Comment:
Should comment what the various indexes are.  I think it's label name -> label value -> slice of SeriesRef.